app/Resources/views/themes/ezplatform/line/bundle_card_old.html.twig

```diff
@@ -0,0 +1,54 @@
+<div class="bundle-card-line-container{% if extra.page is defined %} page-{{ extra.page }}{% endif %}">
```

Comment:
I don't think this file should be in this PR ;)

Reply:
This is why we have git. So we can restore the previous version of the file if we need to.
There is no need to keep an unused file in the repo.

Reply:
Ok, I removed this template